### PR TITLE
Add service nodes + stop/start

### PR DESCRIPTION
**Pull request by akegaviar**
Added service nodes and stopping/starting nodes.

### CODE REVIEW

File: docs/platform/view-service-nodes.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+1. Click your project.
+1. Click your network.
+1. Click **Service nodes**.
+1. Click the node name.
+
+Corda service nodes:
+
+* Notary — a [notary node](/blockchains/corda#notary-service).
+* NMS — a [network map service node](/blockchains/corda#network-map-service).
```
Comment (easeev) [inline docs/platform/view-service-nodes.md:25]:
```suggestion
* Network Map Service — a [network map service node](/blockchains/corda#network-map-service).
```

Comment (akegaviar) [reply docs/platform/view-service-nodes.md:25]: Are we going to change `NMS` to `Network Map Service` in the platform UI as well? (currently it's `NMS`.)

Comment (easeev) [reply docs/platform/view-service-nodes.md:25]: Yes, it should be done already (perhaps you have an old network)

@CaJIbHuK should we rename all old NMS to Network Map Service for consistency?

Comment (akegaviar) [reply docs/platform/view-service-nodes.md:25]: > Yes, it should be done already (perhaps you have an old network)

I deployed a new one to test before asking. It's `NMS` on fresh deployments.

Comment (CaJIbHuK) [reply docs/platform/view-service-nodes.md:25]: Sorry, I've created a branch but forgot to create a PR 🤦‍♂️
https://github.com/chainstack/core/pull/2034

**Pull request merged**